### DISCUSSION TITLE
Use GitHub Discussions (rather than repo documents) for proposals

### DISCUSSION THREAD

**Discussion by reece** [HGVSnomenclature/hgvs-nomenclature#5] [HVNC operations]
The repo README currently suggests that we create docs in the repo in order to make proposals. Typically, proposals will merit discussion, so would likely be better served as topics in the Discussions forum.

Proposal: Replace the current README instructions with instructions to raise new proposals in the Discussion tab.

**Comment by jfjlaros** [comment]
The reasoning behind this is to have the latest draft of the proposal online for anyone to see and for all members of the committee to edit. I would indeed also expect people to open a discussion.

The current instructions are only there to show people how to create files and to give the repository a bit of structure. Please feel free to add some procedural guidelines.

**Reply by reece** [reply]
Yes, added guidelines in #6 . This issue and #6 are a pair: Roughly, the request is to not do what's proposed in the README, and instead do what's in #6.

IMO, we should have only one repo with contents that is the nomenclature because that is our primary artifact that we oversee for community. Then, we use discussion threads for community input and issues to track changes that will show up in the nomenclature.

**Reply by ahwagner** [reply]
> IMO, we should have only one repo with contents that is the nomenclature because that is our primary artifact that we oversee for community. Then, we use discussion threads for community input and issues to track changes that will show up in the nomenclature.

I agree with ☝️ this way of managing community input through technical documentation.

**Reply by jfjlaros** [reply]
Yes, that would be preferable indeed. We could propose to merge this repository with [VarNomen](https://github.com/HGVSnomenclature/VarNomen)?

Still, I think it would be good to facilitate working on drafts in a version control system. It is nice to have the latest version, its history and comments (including links to discussions) in one place.

**Reply by ahwagner** [reply]
> I think it would be good to facilitate working on drafts in a version control system. It is nice to have the latest version, its history and comments (including links to discussions) in one place.

@jfjlaros I think so too–but perhaps we could create and maintain Google Docs for this purpose? Those will have auto-versioning (including named version features) and as a WYSIWYG collaborative drafting tool it will be less of a technical burden for community members. This would then serve as a nice basis for a PR when we get to the technical drafting stage.

**Reply by jfjlaros** [reply]
I would not be in favour of using an separate system with separate accounts and a separate history. I see your point about WYSIWYG tools though.

**Reply by ifokkema** [reply]
Although initially, I thought about Google Docs as well, it doesn't allow discussions very well, and discussions can get "lost" the moment the text is adapted.

Perhaps we could move discussions to PRs that are made to submit a proposal? It would require new proposals not to be made in `main` but instead in a feature branch. The PR to incorporate the proposal into `main` could then serve as a place for relevant discussion.

**Comment by reece** [comment]
It looks to me that this discussion is stale. From this thread, other threads, and de facto practice, here's what I think we're doing:

- We have one repo (hgvs-nomenclature). It contains the HGVS Nomenclature web pages, which is our primary product for the community.  The VarNomen name and repo are deprecated.
- GitHub Discussions is being used to raise new features and to discuss usage
- GitHub Issues will be used to track problems and tasks related directly the nomenclature
- Google Docs might be used incidentally, but isn't part of our current practice

I'll close this thread, but we can reopen if you think there's still something to discuss.
